Applications/ApplicationsLib/ProjectData.cpp

```diff
@@ -338,6 +339,13 @@ void ProjectData::parseProcesses(BaseLib::ConfigTree const& processes_config,
                         "dimension");
             }
         }
+        else if (type == "DensityDrivenFlow")
```

Comment:
I think Thermohaline will eventually be implemented. Let us decide together with @waltherm